manifests/clearcache.pp

```diff
@@ -0,0 +1,25 @@
+# = Define: composer::clearcache
+#
+# == Parameters:
+#
+# [*home_dir*]
+#   Home directory (optional).
+#
+# == Example:
+#
+# ::composer::clearcache { 'vagrant': }
+#
+define composer::clearcache($home_dir = undef) {
```

Comment:
``clearcache`` is hard to read and therefore I'd rename this to ``clear_cache``.